packages/SwingSet/src/kernel/kernel.js

```diff
@@ -283,6 +285,36 @@ export default function buildKernel(kernelEndowments, kernelOptions = {}) {
     }
   }
 
+  function removeVatManager(vatID, reason) {
+    const old = ephemeral.vats.get(vatID);
+    ephemeral.vats.delete(vatID);
+    const err = reason ? Error(reason) : undefined;
+    old.notifyTermination(err); // XXX TODO: most of the places where notifyTermination gets passed don't need it
```

Comment:
let's walk through all the places this reaches, I want to make sure that a kernel stack trace doesn't make it into a vat-admin `.done` rejection